src/planscape/plan/views.py

```diff
@@ -297,42 +301,39 @@ def get_project_areas(request: HttpRequest) -> HttpResponse:
 
 def get_scores(request: HttpRequest) -> HttpResponse:
     try:
-        plan = get_plan_by_id(request.GET)[0]
-        geo = plan.geometry
-        reg = plan.region_name.removeprefix('RegionName.').lower()
-
-        accumulator = RasterPixelAccumulator(geo)
-
-        ids_to_conditions = {
-            c.pk: c.condition_name
-            for c in BaseCondition.objects.filter(region_name=reg).all()}
-        raster_names_to_ids = {
-            c.raster_name: c.condition_dataset.pk
-            for c in Condition.objects.filter(
-                condition_dataset_id__in=ids_to_conditions.keys()).filter(
-                is_raw=False).all()}
-
-        for r in ConditionRaster.objects.filter(
-                name__in=raster_names_to_ids.keys()).filter(
-                raster__bboverlaps=accumulator.geo).all():
-            id = raster_names_to_ids[r.name]
-            condition = ids_to_conditions[id]
-            accumulator.process_raster(
-                r.raster, condition)
-
-        conditions = []
-        for c in accumulator.stats.keys():
-            if accumulator.stats[c]['count'] == 0:
-                conditions.append({'condition': c})
-            else:
-                conditions.append(
-                    {'condition': str(c), 'mean_score': str(accumulator.stats[c]
-                     ['sum'] / accumulator.stats[c]['count'])})
-
-        response = {'conditions': json.dumps(conditions)}
-        return HttpResponse(
-            JsonResponse(response),
-            content_type='application/json')
+        with connection.cursor() as cursor:
+            plan = get_plan_by_id(request.GET)[0]
```

Comment:
can you add user validation here to check that the user is allowed to access the plan? i have a PR that does this that ill send out for review soon with examples!

Reply:
I see #354.
I'll submit this PR as-is and follow up with user validation once #354 has been submitted.